src/renderer/store/modules/playlists.js

```diff
@@ -79,15 +81,30 @@ const actions = {
 
     try {
       await DBPlaylistHandlers.create([payload])
+
+      const noQuickBookmarkSet = !rootState.settings.quickBookmarkTargetPlaylistId || !state.playlists.find((playlist) => playlist._id === rootState.settings.quickBookmarkTargetPlaylistId)
```

Comment:
As you don't need the value and only need to know if it exists, you could use `Array#some`.

```suggestion
      const noQuickBookmarkSet = !rootState.settings.quickBookmarkTargetPlaylistId || !state.playlists.some((playlist) => playlist._id === rootState.settings.quickBookmarkTargetPlaylistId)
```

Reply:
Good call-out, not sure what I was thinking. I might've copy-pasted it from another context without thinking about it.